src/sync/sync-state.js

```diff
@@ -49,6 +49,15 @@ export class SyncState extends TypedEmitter {
     }
   }
 
+  /**
+   * @param {string} peerId
+   */
+  disconnectPeer(peerId) {
```

Comment:
Nit: should this be called `removePeer()` to match `addPeer()`? Same comment applies elsewhere.

Reply:
I renamed it because there is a `removePeer` in CoreSyncState which means something different.